src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -92,44 +101,55 @@ internal class OktaApi : IOktaApi {
 		string factorId,
 		string challengeResponse
 	) {
-		var request = new VerifyMfaChallengeResponseRequest(
-			StateToken: stateToken,
-			PassCode: challengeResponse );
-
-		var resp = await _httpClient.PostAsJsonAsync(
-			$"authn/factors/{factorId}/verify",
-			request,
-			SourceGenerationContext.Default.VerifyMfaChallengeResponseRequest );
-
-		var authnResponse = await JsonSerializer.DeserializeAsync(
-			await resp.Content.ReadAsStreamAsync(),
-			SourceGenerationContext.Default.AuthenticateResponseRaw );
-		if( authnResponse?.SessionToken is not null ) {
-			return new AuthenticateResponse.Success( authnResponse.SessionToken );
+		try {
+			var request = new VerifyMfaChallengeResponseRequest(
+				StateToken: stateToken,
+				PassCode: challengeResponse );
+
+			var resp = await _httpClient.PostAsJsonAsync(
+				$"authn/factors/{factorId}/verify",
+				request,
+				SourceGenerationContext.Default.VerifyMfaChallengeResponseRequest );
+
+			var authnResponse = await JsonSerializer.DeserializeAsync(
+				await resp.Content.ReadAsStreamAsync(),
+				SourceGenerationContext.Default.AuthenticateResponseRaw );
+			if( authnResponse?.SessionToken is not null ) {
+				return new AuthenticateResponse.Success( authnResponse.SessionToken );
+			}
+			throw new BmxException( "Okta MFA challenge response session token null" );
+		} catch( Exception ex ) {
+			throw new BmxException( "Error verifying Okta MFA challenge response", ex );
 		}
-		throw new BmxException( "Error verifying Okta MFA challenge response" );
 	}
 
 	async Task<OktaSession> IOktaApi.CreateSessionAsync( string sessionToken ) {
-		var resp = await _httpClient.PostAsJsonAsync(
-			"sessions",
-			new CreateSessionRequest( sessionToken ),
-			SourceGenerationContext.Default.CreateSessionRequest );
-
-		var session = await JsonSerializer.DeserializeAsync(
-			await resp.Content.ReadAsStreamAsync(),
-			SourceGenerationContext.Default.OktaSession );
-
-		return session ?? throw new BmxException( "Error creating Okta session" );
+		try {
+			var resp = await _httpClient.PostAsJsonAsync(
+				"sessions",
+				new CreateSessionRequest( sessionToken ),
+				SourceGenerationContext.Default.CreateSessionRequest );
+
+			var session = await JsonSerializer.DeserializeAsync(
+				await resp.Content.ReadAsStreamAsync(),
+				SourceGenerationContext.Default.OktaSession );
+			return session ?? throw new BmxException( "Creating Octa Session returned null" );
```

Comment:
```suggestion
			return session ?? throw new BmxException( "Creating Okta Session returned null" );
```

Also your title has it as Octa